src/templates/pageComponentsV2/PageBody.jsx

```diff
@@ -1,6 +1,6 @@
 export const PageBody = ({ chunk }) => {
   return (
-    <div className="col is-8 is-offset-1-desktop is-12-touch print-content page-content-body">
+    <div className="col is-offset-1-desktop is-12-touch print-content page-content-body">
```

Comment:
assuming the deletion of `is-8` here is also replicated onto templates? also, could you check if this leads to teh side nav cutting into content?

Reply:
Hmm no, this is not a replication of the template. The size of the preview is too small with this `is-8` class, because the browser uses the entire screen width in its calculation instead of only the width of the preview. I opted to remove this so that the preview does not look distorted.